src/cpu/o3/issue_queue.cc

```diff
@@ -186,7 +186,7 @@ IssueQue::checkScoreboard(const DynInstPtr& inst)
                 panic("dst is not load");
             }
             scheduler->loadCancel(dst_inst);
-            DPRINTF(Schedule, "[sn %lu] %s can't get data from bypassNetwork, dst inst: %s\n",
+            DPRINTF(Schedule, "[sn:%llu] %s can't get data from bypassNetwork, dst inst: %s\n",
```

Comment:
I actually did this on purpose because this can help us find scheduler‘s log from a log irrelevant information easier and faster.
It is very useful in scheduler debugging.

Reply:
ok